src/udev/udev-builtin-input_id.c

```diff
@@ -240,8 +240,12 @@ static bool test_pointers(struct udev_device *dev,
                         is_touchscreen = true;
         }
 
-        if (has_rel_coordinates && has_mouse_button)
-                is_mouse = true;
+        if (has_mouse_button) {
+                if (has_rel_coordinates)
+                        is_mouse = true;
+                else if (!has_abs_coordinates) /* mouse buttons and no axis */
+                        is_mouse = true;
```

Comment:
This can be written as:
```C
if (has_mouse_button &&
    (has_rel_coordinates || !has_abs_coordinates /* mouse_buttons and no axis */))
      is_mouse = true;
```